contracts/DEX.sol

```diff
@@ -0,0 +1,25 @@
+//SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.2;
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract DEX {
+    event CDTToSSVConverted(uint256 amount);
+
+    IERC20 public cdtToken;
+    IERC20 public ssvToken;
```

Comment:
if we want this contract to be generic you should rename the tokens variables to be `sourceToken` and `destinationToken`/`targetToken`.

Reply:
i think no, Lior wants to have two ways swap